p2p/src/peer_manager/mod.rs

```diff
@@ -68,6 +73,11 @@ const PEER_MGR_HEARTBEAT_INTERVAL_MIN: Duration = Duration::from_secs(5);
 /// Upper bound for how often [`PeerManager::heartbeat()`] is called
 const PEER_MGR_HEARTBEAT_INTERVAL_MAX: Duration = Duration::from_secs(30);
 
+/// How often send ping requests to peers
+const PEER_MGR_PING_CHECK_PERIOD: Duration = Duration::from_secs(60);
+/// When a peer is detected as dead and disconnected
+const PEER_MGR_PING_TIMEOUT: Duration = Duration::from_secs(150);
```

Comment:
This should be controllable through command line arguments in the future. Maybe not necessarily now. But this is definitely going into functional tests in the future.

Reply:
OK. I added command line arguments.